export/export.go

```diff
@@ -107,28 +107,46 @@ func NewExportService(client clientset.Interface, pods PodService, nodes NodeSer
 	}
 }
 
+type options struct {
+	ignoreErr bool
+}
+
+type ignoreErrOption bool
+
+type Option interface {
+	apply(*options)
+}
+
+func (i ignoreErrOption) apply(opts *options) {
+	opts.ignoreErr = bool(i)
+}
+
+func (s *Service) WithIgnoreErr() Option {
+	return ignoreErrOption(true)
+}
+
 // Get gets all resources from each service.
-func (s *Service) get(ctx context.Context) (*ResourcesForExport, error) {
+func (s *Service) get(ctx context.Context, opts options) (*ResourcesForExport, error) {
 	errgrp := util.NewErrGroupWithSemaphore(ctx)
 	resources := ResourcesForExport{}
 
-	if err := s.listPods(ctx, &resources, &errgrp); err != nil {
+	if err := s.listPods(ctx, &resources, &errgrp, opts); err != nil {
 		return nil, xerrors.Errorf("call listPods: %w", err)
 	}
-	if err := s.listNodes(ctx, &resources, &errgrp); err != nil {
+	if err := s.listNodes(ctx, &resources, &errgrp, opts); err != nil {
 		return nil, xerrors.Errorf("call listNodes: %w", err)
 	}
-	if err := s.listPvs(ctx, &resources, &errgrp); err != nil {
+	if err := s.listPvs(ctx, &resources, &errgrp, opts); err != nil {
 		return nil, xerrors.Errorf("call listPvs: %w", err)
 	}
-	if err := s.listPvcs(ctx, &resources, &errgrp); err != nil {
+	if err := s.listPvcs(ctx, &resources, &errgrp, opts); err != nil {
 		return nil, xerrors.Errorf("call listPvcs: %w", err)
 	}
-	if err := s.listStorageClasses(ctx, &resources, &errgrp); err != nil {
+	if err := s.listStorageClasses(ctx, &resources, &errgrp, opts); err != nil {
 		return nil, xerrors.Errorf("call listStorageClasses: %w", err)
 	}
-	if err := s.listPcs(ctx, &resources, &errgrp); err != nil {
-		return nil, xerrors.Errorf("call listPcs: %w", err)
+	if err := s.listPcs(ctx, &resources, &errgrp, opts); err != nil {
+		return nil, xerrors.Errorf("call Pcs: %w", err)
```

Comment:
This change looks mistake, maybe? I don't see any reason for this. (because other formats of error are `call listHoge: %w`)